cmd/kueue/main.go

```diff
@@ -160,6 +160,9 @@ func main() {
 		cacheOptions = append(cacheOptions, cache.WithExcludedResourcePrefixes(cfg.Resources.ExcludeResourcePrefixes))
 		queueOptions = append(queueOptions, queue.WithExcludedResourcePrefixes(cfg.Resources.ExcludeResourcePrefixes))
 	}
+	if cfg.FairSharing != nil {
+		cacheOptions = append(cacheOptions, cache.WithFairSharing(cfg.FairSharing.Enable))
```

Comment:
I'm wondering what is the best way to cover this with a test. We already have integration tests, but it slipped. Maybe it is a sign we should have an integration test suite for fair sharing, wdyt @alculquicondor ? In order to merge this quickly I'm ok with adding the test as a follow up.

Reply:
It slipped because the integrations test don't use the main Kueue config,  an e2e test can cover this.

Reply:
Oh yes, I meant e2e

Reply:
Right, let's add an E2E test as a follow up.

Reply:
I opened the issue to keep track of it: https://github.com/kubernetes-sigs/kueue/issues/2431